src/AVRouting/AVCustomDeviceRoute.cs

```diff
@@ -0,0 +1,21 @@
+#if IOS
+//
+// AVCustomDeviceRoute.cs: AVCustomDeviceRoute Complementing methods
+//
+// Author:
+//   Manuel de la Pena <mandel@microsoft.com>
+//
+using System;
+using Network;
+
+#nullable enable
+
+namespace AVRouting {
+	partial class AVCustomDeviceRoute {
+
+		 public NWEndpoint NetworkEndpoint => new NWEndpoint (_NetworkEndpoint, owns: true);
```

Comment:
I don't think you own the `_NetworkEndpoint` handle, since the selector doesn't match any of the special selector names that indicate that the return value is retained (https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/mmRules.html#//apple_ref/doc/uid/20000994-BAJHFBGH):

```suggestion
		 public NWEndpoint NetworkEndpoint => new NWEndpoint (_NetworkEndpoint, owns: false);
```

Reply:
@rolfbjarne Okay, I have added the unit test and changed the owns to false!

Reply:
@tj-devel709 did you forget to push? no new commits here...

Reply:
@rolfbjarne Yes I did.. haha sorry, should be up now!